src/server/api/boards/boards.js

```diff
@@ -100,6 +100,47 @@ function addCardToColumn(board, columnId, cardName) {
   });
 }
 
+function validateColumnId(board, columnId, newColumnId) {
+  let oldFound = false;
+  let newFound = false;
+
+  board.columns.forEach(function(element) {
+    if (element._id.toString() === columnId.toString()) {
+      oldFound = true;
+    }
+    if (element._id.toString() === newColumnId.toString()) {
+      newFound = true;
+    }
+  });
+  return oldFound && newFound;
```

Comment:
also you could use [Array.prototype.**some**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some). it might not be the most efficient way though.

```js
const oldFound = board.columns.some( . . . );
const newFound = board.columns.some( . . . );
return oldFound && newFound;
```